### ISSUE TITLE
Test if config samples are valid trough StyleCI API

### ISSUE THREAD

**Issue by Soullivaneuh** [Soullivaneuh/php-cs-fixer-styleci-bridge#68]
See comment: https://github.com/Soullivaneuh/php-cs-fixer-styleci-bridge/pull/67#issuecomment-225041948

**Comment by Soullivaneuh** [comment]
@GrahamCampbell about this, not sure this is the place for your API but it would be great to be able to download a config reference of `.styleci.yml` file with **all** the possible keywords.

Because the current problem is: If you add a config keyword, the bridge will throw an error because it does not know this one.

If I can integrate the config reference under the tests, I'll be warned of the issue very quickly thanks to Travis.

What do you think?

**Comment by GrahamCampbell** [comment]
I think there is one in the docs.

**Comment by Soullivaneuh** [comment]
Yeah... but I was thinking about something more... automatic? :-)
